delete-site.sh

```diff
@@ -1,16 +1,25 @@
 #!/bin/bash
+DOMAIN=$2
+USER=$1
 
 service nginx stop
+service php7.0-fpm stop
 
-rm -rf /var/www/$2
+rm -rf /var/www/$DOMAIN
```

Comment:
Remove everything without any confirmation whatsoever? What could possibly go wrong?

Considering how destructive this operations are, I'd rather put everything under **several** confirms and probably typing some random string.